cla_public/static-src/javascripts/modules/address-finder.js

```diff
@@ -73,7 +73,7 @@
       this.addresses = data;
 
       if (data.length < 1) {
-        this.showError('No addresses were found with that postcode');
+        this.showHelp('No addresses were found with that postcode, but you can still enter your address manually');
```

Comment:
OK there are three lines of content in that JS file. We should probably use underscore templates, as with other translated elements on the same page. I'll do that tomorrow.